### PR TITLE
Add the download util in AutoMM

### PR DESCRIPTION
**Pull request by zhiqiangdon**
1. Add the download function in AutoMM's utils.
2. Remove the duplicates from `examples` and the unit tests.

### CODE REVIEW

File: multimodal/setup.py [hunk 1]
```diff
@@ -25,6 +25,8 @@
     'Pillow',
     'tqdm',
     'boto3',
+    'requests',
+    'uuid',
```
Comment (sxjscience) [inline multimodal/setup.py:29]: No need to add uuid I think.

**Review by sxjscience — APPROVED** [review]
LGTM

**Comment by szha** [comment]
Job PR-1931-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1931/2/index.html

**Review by bryanyzhu — APPROVED** [review]
LGTM, thanks.

**Pull request merged**